### ISSUE TITLE
git-summary output format problem.

### ISSUE THREAD

**Issue by ZJUGuoShuai** [tj/git-extras#694]
I use Fedora 27. And I encountered this problem:
```
➜  Find git:(backend) git summary

 project  : Find
 repo age : 10 weeks
 active   : 13 days
 commits  : 47
 files    : 11
 authors  : 
47\x09Xiaohei  100.0%
```
The last line: **47\x09Xiaohei  100.0%**

The '\t' didn't come out correctly.

My git-extras version information:
```
Name         : git-extras
Version      : 4.4.0
Release      : 2.fc27
Arch         : noarch
Size         : 669 k
Source       : git-extras-4.4.0-2.fc27.src.rpm
Repo         : @System
From repo    : fedora
```

**Comment by spacewander** [comment]
@ZJUGuoShuai 
The author section is generated mainly by `git shortlog -n -s`.
What is the output of `git shortlog -n -s` in your repo?

**Comment by ZJUGuoShuai** [comment]
It outputs this:
`    50  Xiaohei`

![output screencast](https://wx3.sinaimg.cn/mw1024/8163951egy1fo8wifhjmlj20ki06gt8o.jpg)

**Comment by ZJUGuoShuai** [comment]
@spacewander Thanks a lot. That's really a problem. The reason is that there is a '\t' in the output of `git shortlog -n -s`, and `column` utility will escape tab if a separator is specified.

**Comment by spacewander** [comment]
@ZJUGuoShuai 
Fixed in the commit d4bf79df8f329af8cd8ad2aa50a1fd60e0188bb0
